production/helm/loki-stack/templates/_helpers.tpl

```diff
@@ -31,6 +31,37 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Common labels
+*/}}
+{{- define "loki-stack.labels" -}}
+helm.sh/chart: {{ include "loki-stack.chart" . }}
+{{ include "loki-stack.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end -}}
+
+{{/*
+Selector labels
+*/}}
+{{- define "loki-stack.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "loki-stack.name" . }}
```

Comment:
is it necessary to change `app` and `release` labels ? I would feel more confident if we could keep the same default label set. The alternative would be to update promtail relabeling config to keep the same labeling.

Reply:
The changes just reflect the Helm best practices / Kubernetes standard. 
See: 
* https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
* https://helm.sh/docs/topics/chart_best_practices/labels/

Using them makes it easier to "guess" the right label when using the chart.

Reply:
I'm mostly concerned about the fact that we have a relabelling config relying on the `app` label to build the job stream label.

And since this now it's gone it will fallback on the pod name which seems to be a different value than before (e.g. `{{ template "loki-stack.fullname" . }}`).

I think we should keep the `app` label for consistency.  WDYT about adding it back and removing changes in the test.sh ?

Reply:
Using standard labels was the main reason for this pr.

I've added the app label additionally for now.

How about changing the relabeling config too, to get rid of the old label?

I guess you're talking about this: https://github.com/grafana/loki/blob/1b251e54a48e0b5861d1e3f1d601ba90b5b9ba48/production/helm/promtail/templates/configmap.yaml

If so, i could do the changes in this pr too.

Reply:
@slim-bean WDYT ?

Reply:
I think we prefer to keep the label as they are. You can add more if you want.

Reply:
So it's ok as its is right now as the app label was added again?

Reply:
Yes I just need to test this locally with upgrade.